ods_ci/run_robot_test.sh

```diff
@@ -317,19 +329,30 @@ echo "${exit_status}"
 
 # send test artifacts by email
 if ${EMAIL_REPORT}
- then
-     tar cvzf rf_results.tar.gz ${TEST_ARTIFACT_DIR} &> /dev/null
+  then
+     tar cvzf rf_results.tar.gz "${TEST_ARTIFACT_DIR}" &> /dev/null
      size=$(du -k rf_results.tar.gz | cut -f1)
      if [ "${size}" -gt 20000 ]
         then
             echo "Test results artifacts are too large for email"
             rm rf_results.tar.gz
-            tar cvzf rf_results.tar.gz $(find ${TEST_ARTIFACT_DIR} -regex  '.*\(xml\|html\)$') &> /dev/null
+            tar cvzf rf_results.tar.gz $(find "${TEST_ARTIFACT_DIR}" -regex  '.*\(xml\|html\)$') &> /dev/null
      fi
-     python3 ods_ci/utils/scripts/Sender/send_report.py send_email_report -s ${EMAIL_FROM} -r ${EMAIL_TO} -b "ODS-CI: Run Results" \
-                        -v ${EMAIL_SERVER} -a "rf_results.tar.gz" -u  ${EMAIL_SERVER_USER}  -p  ${EMAIL_SERVER_PW} \
-                        -l ${EMAIL_SERVER_SSL} -d ${EMAIL_SERVER_UNSECURE}
+     python3 ods_ci/utils/scripts/Sender/send_report.py send_email_report -s "${EMAIL_FROM}" -r "${EMAIL_TO}" -b "ODS-CI: Run Results" \
+                        -v "${EMAIL_SERVER}" -a "rf_results.tar.gz" -u "${EMAIL_SERVER_USER}" -p "${EMAIL_SERVER_PW}" \
+                        -l "${EMAIL_SERVER_SSL}" -d "${EMAIL_SERVER_UNSECURE}"
+fi
+
+
+if test "${OPEN_REPORT_IN_BROWSER}" != "false"
+  then
+    if test "${OPEN_REPORT_IN_BROWSER}" != "true"
```

Comment:
how many values do you expect this variable to have? "false", "true" and?

Reply:
imo i'ts ambiguous what value this variable can have. It'd be probably better to use it as boolean and control the default browser with another option

Reply:
https://github.com/red-hat-data-services/ods-ci/pull/936/files#diff-127a3d7ae5cdfa3e5e9cadb9daf76ea4763da79227ceffecd571fb764b93603fR163

Reply:
@lugi0 yeah I saw that comment, but that still sounds ambiguous to me: "if not false (hence true?); if true (is it different from not false?)"

Reply:
I'm not sure I follow your logic; if you give it false, it's not going to enter that block at all, if you give it true, it opens the default browser, if you give it a different string `xyz` it's going to open browser `xyz` (so, valid for valid browsers, fails for random strings). It looks ok to me, or am I missing something?

Reply:
just saying that the variable seem ambiguous to me, it starts as boolean but then you can assign other string values. I know it works, not saying differently. Just ambiguous to me. If it's only me seeing this it's ok to merge as it is, since it works

Reply:
Well, I can update to introduce a new flag for it, but TBH - it seems simple to me to use this one flag directly so that you don't have to remember another flag name for this feature - that was my main motivation to implement it this way.

Reply:
hmm okay, please document it in https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/docs/RUN_ARGUMENTS.md